testutil/testnode/node_interaction_api.go

```diff
@@ -163,6 +163,10 @@ func (c *Context) PostData(account, broadcastMode string, ns, msg []byte) (*sdk.
 // with squaresize >= 2. TODO: perform checks (is a power of 2 and is > 2) on
 // the passed squaresize arg
 func (c *Context) FillBlock(squareSize int, accounts []string, broadcastMode string) (*sdk.TxResponse, error) {
```

Comment:
Sorry, Github won't let me provide a suggestion for the doc comment above this function but I think the TODO should be removed. Proposal to use:

```go
// FillBlock creates and submits a single transaction that is large enough to
// create a square of the desired size. broadcast mode indicates if the tx
// should be submitted async, sync, or block. (see flags.BroadcastModeSync). If
// broadcast mode is the string zero value, then it will be set to block. 
func (c *Context) FillBlock(squareSize int, accounts []string, broadcastMode string) (*sdk.TxResponse, error) {
...
```

Reply:
The sentence above the TODO can also be removed because this is no longer accurate: `This function does not perform checks on the passed squaresize arg, and only works`

Reply:
Removed given sentence ^^

Reply:
Thanks for removing. I'm okay with the current PR or adding the line you mentioned.